scripts/output/comparison/checkClimatePercentiles.R

```diff
@@ -0,0 +1,96 @@
+# |  (C) 2006-2023 Potsdam Institute for Climate Impact Research (PIK)
+# |  authors, and contributors see CITATION.cff file. This file is part
+# |  of REMIND and licensed under AGPL-3.0-or-later. Under Section 7 of
+# |  AGPL-3.0, you are granted additional permissions described in the
+# |  REMIND License Exception, version 1.0 (see LICENSE file).
+# |  Contact: remind@pik-potsdam.de
+#
+# This script is used to check the temperature outcomes of MAGICC when run inside REMIND
+#
+# The In-GAMS version of MAGICC can only run only one configuration (that is, parametrization) of MAGICC at a time, 
+# picked in cfg$climate_assessment_magicc_prob_file_iteration. The available configuration files were designed to 
+# roughly reproduce a certain quantile outcome of 2100 temperature for scenarios that are as similar as possible to 
+# an RCP (hence the nomenclature of the config files e.g. RCP20_50.json will be close to the 50th percentile outcome in 
+# a RCP2.0-like scenario). This quantile outcome is of course not guaranteed, so this script helps check the validity of
+# the assumed quantile against the 600-member MAGICC7 AR6 assessment that is run by default at the end of each run. 
+# Therefore, for it to work, the output script MAGICC7_AR6.R must have already run and added the `MAGICC7 AR6|**` 
+# variables to `REMIND_generic_*.mif`. To verify the output, check the figures in `check_output_figures.pdf` and the 
+# table in `check_output_summary.csv`
+#
+require(purrr)
+require(stringr)
+require(quitte)
```

Comment:
We're using too much of the tidyverse here, so in this particular case I think it's a lot simpler to load it all. I think the only one not being used here is `lubridate`.